src/main/java/com/es/phoneshop/model/product/RecentlyViewedProducts.java

```diff
@@ -0,0 +1,46 @@
+package com.es.phoneshop.model.product;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class RecentlyViewedProducts {
```

Comment:
Rename to something more sensitive: is it a service(RecentlyViewedProductsService)/dao?
Should it implement interface?